concrete/controllers/single_page/dashboard/users/add.php

```diff
@@ -94,9 +63,6 @@ public function submit()
             $response = $validator->validateSaveValueRequest(
                 $controller, $this->request, $uak->isAttributeKeyRequiredOnRegister()
             );
-            /**
-             * @var $response ResponseInterface
```

Comment:
This should stay as well unless `validateSaveValueRequest` gets a proper docblock